tests/integration/conftest.py

```diff
@@ -1204,6 +1214,17 @@ def last_invocation_queue_job_id(self):
         assert self._last_invocation_queue_job_id
         return self._last_invocation_queue_job_id
 
+    @property
+    def metadata(self):
+        """A dictionary of arbitrary metadata for this runner. Useful for when you want to attach extra fixture data
+        to the runner.
+
+        Returns:
+            dict
+        """
+
+        return self._metadata
```

Comment:
Fun new feature that allows the test to attach arbitrary data to the runner. Neat!